src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementBannerEntityUpdater.java

```diff
@@ -194,7 +194,7 @@ private void updateManagementBanner(DefaultApi api, OrganizationId organizationI
   }
   
   private void updateFeaturedMedia(OrganizationId organizationId, DefaultApi api, BannerId bannerId, Integer featuredMedia) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "updateFeaturedMedia" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)